ruby_on_rails/testing/model_testing.md

```diff
@@ -0,0 +1,132 @@
+### Introduction
+
+The last lesson covered unit testing and now we will move into model testing. Here you will get an idea of how to test within Rails for model testing as well as a deeper dive of how to structure these tests.
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+*   What are model tests?
+*   How do I run a model test?
+*   How to structure model testing?
+
+## What are model tests?
+
+A model test is a unit test in the context of Ruby on Rails. Model tests in this case are designed to test the business logic of your models. Let's say you need to test a custom validation method for your blog post. In this case you would need to reach into the model aspect of your testing skillset. Before you write the RSpec there will be a few tools you will need to familiarize yourself with.
+
+## Setting the test
+
+The first tool that we are going to use to set up the data is factory bot. Factory bot is a gem that you can use to set up test data and create necessary data for your tests. Adding Factory bot is similar to adding other testing gems. Add the following line to the to the following section in your Gemfile.
```

Comment:
I think it would be useful to add a link to Factory Bot, so that people can easily read more about it.
https://github.com/thoughtbot/factory_bot